src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -820,22 +834,31 @@ const FormCreationForm = ({csrftoken, formUuid, formHistoryUrl }) => {
         // Update/create logic rules
         try {
             const {logicRules, logicRulesToDelete} = state;
-            const createdRules = await saveLogicRules(formUrl, csrftoken, logicRules, logicRulesToDelete);
+            const createdRules = await saveLogicRules(
+                formUrl, csrftoken, logicRules, logicRulesToDelete, 'logicRules'
```

Comment:
I would have expected this calls not to be affected - `saveLogicRules` and `savePriceRules` both already have their correct endpoint embedded, so that's the location where I would also include the prefix. :grimacing: